package.json

```diff
@@ -251,7 +251,8 @@
     "esy-oniguruma": "onivim/esy-oniguruma#4698ce4",
     "editor-input": "onivim/editor-input#c494950",
     "isolinear": "revery-ui/isolinear#b17ce17",
-    "revery-terminal": "revery-ui/revery-terminal#2ef1c65"
+    "revery-terminal": "revery-ui/revery-terminal#2ef1c65",
+    "reason-textmate": "onivim/reason-textmate#5269e5f"
```

Comment:
TODO maybe, if we want to do a proper release instead of using a resolution

Reply:
Sure, I'll do a release. Awesome to have support for plist/xml themes and grammars now!

Reply:
Published as `reason-textmate@3.1.0`